grunt/config/copy.js

```diff
@@ -53,7 +53,20 @@ module.exports = function (grunt, options) {
                     filter: function(filepath) {
                         return grunt.config('helpers').includedFilter(filepath);
                     },
-                    flatten: true
+                    
+                    rename: function(destFolder, srcFileName) {
+                        var name = "assets",
```

Comment:
Can you:

Move the function to the top of the file and then reference it where needed, rather than write it multiple times.
``var`` for each line rather than commas.
Rename ``name`` to ``collateAtName``  
Rename ``folder`` to ``collateAtFolder``  
Rename ``endOfFolder`` to ``startOfCollatePath``  
Rename ``re`` to ``testEndsWithCollateName``  
Rename ``output`` to ``collatedFilePath``  
(Or something more descriptive for each one than we've currently got.)